programming/robot_api/comp_mode.md

```diff
@@ -5,4 +5,37 @@ title: Competition Mode
 
 # Competition Mode
 
-TODO
\ No newline at end of file
+
+## What is Comp Mode
+
+During a competition match a Competition Mode USB is inserted into a spare USB port on your robot (See the [kit assembly guide]({{ site.baseurl }}/tutorials/assembly) for details on the spare USB port you need to leave).
+This will instruct your robot that it is in competition mode and will enable a game timeout, disable the WiFi and update certain `robot` attributes.
```

Comment:
Nit: This sentence is a short version of what's explained below. Might be easier to just remove it and pass that off to the section below.

Reply:
1e11100